src/main/java/frc/robot/commands/test_comm/PoseTransformTest.java

```diff
@@ -31,15 +28,15 @@ public PoseTransformTest(SwerveDriveImpl drive) {
     @Override
     public void initialize() {
         startTime = Timer.getFPGATimestamp();
-        finalPose = drive.getPose().transformBy(new Transform2d(
-                new Translation2d(drive.getRequestedXPos(0), drive.getRequestedYPos(0)),
-                new Rotation2d(drive.getRequestedHeading(0))
-        ));
+//        finalPose = drive.getPose().transformBy(new Transform2d(
+//                new Translation2d(drive.getRequestedXPos(0), drive.getRequestedYPos(0)),
+//                new Rotation2d(drive.getRequestedHeading(0))
+//        ));
     }
 
     @Override
     public void execute() {
-        drive.runChassisPID(finalPose.getX(), finalPose.getY(), finalPose.getRotation().getRadians());
+        drive.driveWithPID(finalPose.getX(), finalPose.getY(), finalPose.getRotation().getRadians());
```

Comment:
This references a variable that hasn't been defined, and it will probably throw a NullPointerException. Define finalPose in initialize using:
finalPose = drive.getPose().transformBy(new Transform2d(
     new Translation2d(/*x distance to move*/ 1, /*y distance to move*/ 1),
     Rotation2d.fromDegrees(/*Amount to rotate in degrees*/45)
));
The commented out code doesn't look like it would work.